### PR TITLE
Write files

### PR DESCRIPTION
**Pull request by Guillermo849**

### CODE REVIEW

File: caesar_files.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  end
+
+  private
+
+  attr_reader :name, :text
+end
+
+# It will read the files.
+class FileReader
+  ReadErrros = Class.new(StandardError)
```
Comment (OlekKovalevych) [inline caesar_files.rb:29]: fix typo and name convention

File: caesar_files.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Class that will write the input from the user into a file.
+class FileWriter
+  WriterErros = Class.new(StandardError)
+  NameEmpty = Class.new(WriterErros)
+  FileNotFound = Class.new(WriterErros)
```
Comment (OlekKovalevych) [inline caesar_files.rb:5]:
```suggestion
  EmptyNameError = Class.new(WriterErros)
  FileNotFoundError = Class.new(WriterErros)
```

Comment (Guillermo849) [reply caesar_files.rb:5]: +1

File: caesar_files.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Class that will write the input from the user into a file.
+class FileWriter
+  WriterErros = Class.new(StandardError)
```
[the remark this answers is not in the document]
Comment (Guillermo849) [reply caesar_files.rb]: +1

File: caesar_files.rb (hunk 1 above)
Comment (Guillermo849) [reply caesar_files.rb]: +1

File: file_operations/file_reader.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module FileOperations
+  # It will read the files.
+  class FileReader
```
Comment (OlekKovalevych) [inline file_operations/file_reader.rb:5]:
```suggestion
  class Reader
```

File: runner.rb [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require_relative 'caesar_cipher'
+require_relative 'file_operations/file_reader'
+require_relative 'file_operations/file_writer'
+
+# Class that will ask the user for input for the cipher method of the Caesar class
+# It will repite this process until the user says something other tha y(Yes)
+class Runner
+  include FileOperations
```
Comment (OlekKovalevych) [inline runner.rb:10]:
```suggestion
  ```

**Review by OlekKovalevych — APPROVED** [review]

**Pull request merged**